cmd/bad-key-revoker/main.go

```diff
@@ -288,6 +294,12 @@ type Config struct {
 		// algorithm will wait before retrying in the event of error
 		// or no work to do.
 		BackoffIntervalMax config.Duration `validate:"-"`
+
+		// MaxReplicationLag specifies the minimum duration bad-key-revoker
```

Comment:
bikeshed: "max... specifies the minimum...". The name feels somewhat off. I think using "max" implies that this is somehow a _limit on_ replication lag. I know it's really wordy, but maybe something like "MaxExpectedReplicationLag", or just "ExpectedReplicationLag", or even just "ProcessingDelay"?

Reply:
That makes sense. I feel like keeping this wordy, so as to make it as clear as possible for operators.